client/src/pages/Results.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState, useEffect } from "react";
+import { BrowserRouter } from "react-router-dom";
+import GlobalStyles from "../GlobalStyles";
+import styled from "@emotion/styled";
+import SearchHeader from "../components/SearchHeader";
+import Footer from "../components/Footer";
+import List from "../components/List";
+import { fetchResults } from "../api/results";
+import LocationSrc from "../assets/images/secondBite_Icon_Location.svg";
+import DateSrc from "../assets/images/secondBite_Icon_Date.svg";
+import TimeSrc from "../assets/images/secondBite_Icon_Time.svg";
+import ArrowSrc from "../assets/images/secondBite_Icon_ArrowRight.svg";
+import fruitsSrc from "../assets/images/sb_icon_fruits.svg";
```

Comment:
`sb_icon_fruits` and `fruitsSrc` have different naming conventions like the other sources.

Reply:
Good point. Was afraid to have way too long names, will give them a change.

Reply:
Fix #46